sh.fhs.ksre.yaml

```diff
@@ -0,0 +1,124 @@
+app-id: sh.fhs.ksre
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: ksre
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: '23.08'
+    autodelete: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --socket=pulseaudio
+  - --persist=.renpy
+  - --filesystem=xdg-run/discord-ipc-0
+cleanup:
+  - /include
+modules:
+  - python3-modules.yaml
+  - name: pygame_sdl2
+    buildsystem: simple
+    build-options:
+      arch:
+        aarch64:
+          env:
+            RENPY_DEPS_INSTALL: /app::/usr::/lib::/lib64::/lib/aarch64-linux-gnu
+            CFLAGS: -fcommon
+        x86_64:
+          env:
+            RENPY_DEPS_INSTALL: /app::/usr::/lib::/lib64::/lib/x86_64-linux-gnu
+            CFLAGS: -fcommon
+    build-commands:
+      - python setup.py install --root="/" --prefix="${FLATPAK_DEST}" --optimize=1
+      - mv "${FLATPAK_DEST}/include/python3.11/pygame_sdl2" "${FLATPAK_DEST}/include/pygame_sdl2"
```

Comment:
you should calculate python3.11 part otherwise it's going to break when updating the runtime.

Reply:
`$(python3 -c 'import sys; print("{}.{}".format(*sys.version_info))')`

Reply:
thanks!

it won't be needed anymore tho, moved to renpy official binary build to speed up build time and avoid issues that arose after updating to freedesktop sdk 24.08